### DISCUSSION TITLE
0.3.1 Release

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#78] [Announcements]
See: https://github.com/kubilus1/autoortho/releases/tag/0.3.1-3  for details.

# Important 
Going forward, it's now recommended to use Dokan (https://github.com/dokan-dev/dokany/releases/latest) when installing this in Windows.

Windows execution will now default to using FUSE. WinFSP will continue to work for now, but if you have issues on Windows with WinFSP, try with Dokan.

If there are issues, it is possible to revert back to the raw WinFSP mode by setting 'winfsp_raw = True' in the configuration file.

**Comment by ivotaHR** [comment]
Thank you

**Comment by NNowakowski** [comment]
Will this improve loading speeds on Windows? I've had load times of over 30+ minutes with 0.3.0 on Windows 10

**Comment by kubilus1** [comment]
It should help a bit.  The next release (0.4.0) will be addressing the much
more directly though.

**Comment by mikecolive** [comment]
Do we need to configure something in order to use it with dokan or we just need to install dokan?

**Reply by pntrafewl** [reply]
No, just install it and it should detect it. I think if it doesn't detect it you might have to delete the .autoortho config file in your user directory so it'll write a new one.

**Reply by kubilus1** [reply]
Correct.  It will detect if you have Dokan installed and prefer that over WinFSP.

**Comment by pntrafewl** [comment]
After a few tests at various airports, I gotta say this is very impressive. I'm seeing a big performance gain in VR from the last version and no stutters as of yet. LR needs to put you on the payroll. I'm loving what you are doing here.

**Reply by kubilus1** [reply]
Just wait until the next release.  With some new contributors we are looking at about a 2X speedup.

**Comment by Awemeter** [comment]
Windows security flags this as a virus when ever I download the windows zip.

**Reply by kubilus1** [reply]
Windows Defender is just so helpful!   I just submitted the false positive (https://www.microsoft.com/en-us/wdsi/filesubmission) so hopefully that clears that up.

**Comment by filipto861** [comment]
Hi guys, 

today i updated autoortho to version 0.3.1 and found a few bugs 

Loading time took 30 minutes but that doesnt matter

Once airborn ortho was black from different angles 
![image](https://user-images.githubusercontent.com/120219642/226165415-ad34a46c-7171-4501-b45a-94f12bdc8060.png)

After flying for a couple of minutes ortho wasnt loading anymore
![image](https://user-images.githubusercontent.com/120219642/226165600-7f33cc7a-6186-4bf7-becd-3fef9d5d58b5.png)


Dokan is already installed

**Reply by filipto861** [reply]
![Uploading image.png…]()

**Reply by filipto861** [reply]
![image](https://user-images.githubusercontent.com/120219642/226167914-ef38899a-0215-4703-8258-7d1a65cb7ab8.png)
